cpp/dolfinx/fem/CoordinateElement.h

```diff
@@ -148,17 +156,16 @@ class CoordinateElement
   /// @param[in,out] x The physical coordinates of the reference points X
   /// @param[in] cell_geometry The cell node coordinates (physical)
   /// @param[in] phi Tabulated basis functions at reference points X
```

Comment:
Do we want to add a comment here about the relation between dimensions of the different inputs?
i.e.
```
phi.extent(1) == cell_geometry.extent(0)
phi.extent(0) == x.extent(0)
x.extent(1) == cell_geometry.extent(1)
```
```